BDArmory/UI/BDArmorySetup.cs

```diff
@@ -1784,7 +1784,25 @@ void WindowBDAToolbar(int windowID)
                         ActiveWeaponManager.targetWeightMass.ToString(), leftLabel);
                     priorityLines++;
 
-                    GUI.Label(new Rect(leftIndent, (priorityLines * entryHeight), 85, entryHeight), StringUtils.Localize("#LOC_BDArmory_WMWindow_targetAllies"), leftLabel); //target proximity"
+                    GUI.Label(new Rect(leftIndent, (priorityLines * entryHeight), 85, entryHeight), StringUtils.Localize("#LOC_BDArmory_TargetPriority_TargetDmg"), leftLabel); //target Damage"
+                    if (!NumFieldsEnabled)
+                    {
+                        ActiveWeaponManager.targetWeightDamage =
+                            GUI.HorizontalSlider(
+                                new Rect(leftIndent + (150), (priorityLines * entryHeight), contentWidth - 150 - 38, entryHeight),
+                                ActiveWeaponManager.targetWeightDamage, -10, 10);
+                        ActiveWeaponManager.targetWeightDamage = Mathf.Round(ActiveWeaponManager.targetWeightDamage * 10) / 10;
+                    }
+                    else
+                    {
+                        textNumFields["targetWeightDamage"].tryParseValue(GUI.TextField(new Rect(leftIndent + (90), (priorityLines * entryHeight), contentWidth - 90 - 38, entryHeight), textNumFields["targetWeightDamage"].possibleValue, 4));
```

Comment:
You'll need to add an entry for `targetWeightDamage` to `textNumFields` in `ConfigTextFields` or you'll get an exception.